model/transformers/dffml_model_transformers/ner/ner_model.py

```diff
@@ -177,10 +183,22 @@ class NERModelConfig:
     evaluate_during_training: bool = field(
         "Run evaluation during training at each logging step.", default=False,
     )
+    dataloader_drop_last: bool = field(
+        "Drop the last incomplete batch if the length of the dataset is not divisible by the batch size",
+        default=False,
+    )
+
+    def to_json_string(self):
+        """
+        Serializes this instance to a JSON string.
+        """
+        config_dict = dataclasses.asdict(self)
```

Comment:
Let's use export just in case we ever contain any dffml types

https://intel.github.io/dffml/master/api/util/data.html#dffml.util.data.export

```suggestion
        config_dict = export(self)
```